packages/graphql/src/schema/validation/validate-document.test.ts

```diff
@@ -1215,6 +1215,74 @@ describe("validation 2.0", () => {
                 expect(executeValidate).not.toThrow();
             });
 
+            test("@default on LocalDateTime must be valid datetime correct", () => {
+                const doc = gql`
+                    type User @node {
+                        updatedAt: LocalDateTime @default(value: "2023-07-06T09:45:11.336")
+                    }
+                `;
+
+                const executeValidate = () =>
+                    validateDocument({
+                        document: doc,
+                        additionalDefinitions,
+                        features: {},
+                    });
+
+                expect(executeValidate).not.toThrow();
+            });
+
+            test("@default on Time must be valid datetime correct", () => {
```

Comment:
I would change the test titles as they all ending with `must be valid datetime correct`